docs/source/guides/developing_bits.rst

```diff
@@ -0,0 +1,48 @@
+Developing BITS
+-----------------------
+
+Install for development
```

Comment:
In reST, H1 sections use both underline and overline.

Be certain to use a different underline character for H2 and other section levels.

Reply:
Added a comment for the page title above